scpd/lms/static/sass/partials/base/_variables.scss

```diff
@@ -1,9 +1,10 @@
-// Fonts
-$sans-serif: 'Source Sans Pro', Trebuchet, 'Helvetica Neue', Helvetica, $verdana, sans-serif;
-$f-sans-serif: $sans-serif;
-$serif: 'Crimson Text', Georgia, Cambria, 'Times New Roman', Times, serif;
-$f-serif: $serif;
-$header-sans-serif: $sans-serif;
-
-// Import defaults
+@import 'variable_overrides/palette.scss';
```

Comment:
Shouldn't this be included by `colors.scss`,
since that's the file that actually depends on this?